packages/core/src/components/SortCarets/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import IconCaretUp from '@airbnb/lunar-icons/lib/interface/IconCaretUp';
+import IconCaretDown from '@airbnb/lunar-icons/lib/interface/IconCaretDown';
+import withStyles, { css, WithStylesProps } from '../../composers/withStyles';
+
+export type Props = {
+  // Whether or not to display the bottom caret.
+  down?: boolean;
+  // If enabled, the caret is more pronounced.
+  enableDown: boolean;
+  // If enabled, the caret is more pronounced.
+  enableUp: boolean;
+  // Whether or not to display the top caret.
+  up?: boolean;
+};
+
+/** Carets to indicate sorting on DataTable. */
+export class Carets extends React.Component<Props & WithStylesProps> {
+  static defaultProps = {
+    down: true,
+    enableDown: false,
+    enableUp: false,
+    up: true,
+  };
+
+  renderCaretUp() {
+    const { down, up, enableUp, styles } = this.props;
+
+    return (
+      up && (
+        <span
+          {...css(
+            down && styles.caret_up,
+            styles.caret,
+            enableUp ? this.props.styles.caret_active : styles.caret_inactive,
+          )}
+        >
+          <IconCaretUp size="2em" />
+        </span>
+      )
+    );
+  }
+
+  renderCaretDown() {
+    const { down, up, enableDown, styles } = this.props;
+
+    return (
+      down && (
+        <span
+          {...css(
+            up && styles.caret_down,
+            styles.caret,
+            enableDown ? styles.caret_active : styles.caret_inactive,
+          )}
+        >
+          <IconCaretDown size="2em" />
+        </span>
+      )
+    );
+  }
+
+  render() {
+    const { styles } = this.props;
+
+    return (
+      <span {...css(styles.caret_container)}>
+        {this.renderCaretUp()}
+        {this.renderCaretDown()}
+      </span>
+    );
+  }
+}
+
+export default withStyles(theme => ({
+  caret_container: {
+    marginRight: 0.5 * theme.unit,
+  },
+  caret: {
+    position: 'relative',
+    right: 0.5 * theme.unit,
+  },
+  caret_inactive: {
+    color: theme.color.core.neutral[1],
+  },
+  caret_active: {
+    color: theme.color.core.neutral[4],
+  },
+  caret_up: {
+    bottom: -11,
+  },
+  caret_down: {
+    bottom: 11,
```

Comment:
This number seems arbitrary. `unit * 1.25` or `unit * 1.5`?

Reply:
It is pretty arbitrary unfortunately, I'm trying to get these carets the right distance apart:
<img width="177" alt="Screen Shot 2019-04-22 at 12 10 58 PM" src="https://user-images.githubusercontent.com/8676510/56519756-b6ea0300-64f7-11e9-8ed4-3a9de2833613.png">

So basically it's just a matter of offsetting the padding built into the svg, and not really related to the actual unit in any meaningful way. At `1.25` the gap is too big and at `1.5` it's almost non-existent.

Reply:
May I only issue (which is rare), is when the root font size or unit changes, this scaling will be off. Not sure that will ever happen though.